client/add/add.spec.js

```diff
@@ -9,14 +9,15 @@ Promise.config({warnings: false});
 describe('AddCtrl', () => {
   const errorHandler = (err) => err && console.log(err);
   const BOX_ID = 123;
+  const BOX_ID2 = 1234;
```

Comment:
Ah yeah, I'll change it just so the test is accurate. I blame whoever wrote the first test :-P